addins/Cake.Unicorn.yml

```diff
@@ -1,9 +1,8 @@
 Name: Cake.Unicorn
 NuGet: Cake.Unicorn
-Prerelease: false
 Assemblies:
 - "/**/Cake.Unicorn.dll"
-Repository: https://github.com/georgechang/Cake.Unicorn
+Repository: https://www.nuget.org/packages/Cake.Unicorn/
```

Comment:
@Jericho same comment as above.

Reply:
I already opened an issue to encourage the author to fix several discrepencies I noticed in their nuspec: https://github.com/georgechang/Cake.Unicorn/issues/5